docs/src/app/components/pages/components/app-bar.jsx

```diff
@@ -113,6 +114,16 @@ export default class AppBarPage extends React.Component {
         name="AppBar"
         desc={this.desc}
         componentInfo={this.componentInfo}>
+
+        <Paper style = {{marginBottom: '22px'}}>
+          <CodeBlock>
+          {
+            '//Import statement:\nconst AppBar = require(\'material-ui/lib/app-bar\');\n\n' +
+            '//See material-ui/lib/index.js for more\n'
```

Comment:
Perhaps a better thing to do would be to integrate this with the CodeExample component, but let's keep that for the future (after this release)..